types/koa/index.d.ts

```diff
@@ -716,7 +716,7 @@ declare namespace Application {
         req: IncomingMessage;
         res: ServerResponse;
         originalUrl: string;
-        cookies: Cookies;
+        cookies: typeof Cookies;
```

Comment:
Is this a common pattern? I'm not sure what you're importing here. Drop a link?

Reply:
Line 23 has this:

```ts
import * as Cookies from "cookies";
```

Cookies is a value, not a type (hence the error I'm trying to fix). It is an object representing the exports of the "cookies" module, which has this type:

```ts
{
    default: CookiesStatic;
    get(key: string): string;
    set(key: string, value: any, options?: CookieOptions | undefined): CookiesStatic;
    expire(key: string, options?: CookieOptions | undefined): CookiesStatic;
    defaults: CookieOptions;
    enabled: boolean;
}
```

You can verify that this is correct by reading the [context.cookies section of the koa docs](https://koajs.com/#ctx-cookies-get-name-options-) where it documents `context.cookies.get` and `context.cookies.set`

So, to fix the error that indicated I had was using a value (`Cookies`) where a type was expected, I grabbed the type from the value (using the typescript `typeof` operator) and used that instead.